package/network/utils/iproute2/Makefile

```diff
@@ -122,26 +125,29 @@ define Build/Configure
 endef
 
 TARGET_CFLAGS += -ffunction-sections -fdata-sections -flto
-TARGET_LDFLAGS += -Wl,--gc-sections
+TARGET_LDFLAGS += -Wl,--gc-sections -Wl,--as-needed
```

Comment:
This can be merged into the previous `-Wl` argument, i.e. `-Wl,--gc-sections,--as-needed`

Reply:
Good of you to notice; however "can" and "should" are really different things. I've done that deliberately in the interest of maintainability and avoiding confusion, rather than conflate a rare linker workaround with the multi-line size optimization (`-ffunction-sections -fdata-sections -flto` / `-Wl,--gc-sections`) often used in package Makefiles.

I don't think saving 4 bytes outweighs the above.